Gemfile

```diff
@@ -147,6 +147,7 @@ def override_gem(name, *args)
     dependencies.delete(dependency)
 
     gem name, *args
+    Bundler::UI::Shell.new.warn "Caution, using override_gem for #{name} with args: #{args.inspect}, review Gemfile.dev.rb if this is not desired!"
```

Comment:
> One day I hope to convince @Fryguy to allow me to delete these lines from the Gemfile. Maybe I can convince @jrafanie first then we can team up on @Fryguy 😆

First deprecate it, it's really easy as we should be able to use rubygems' built in deprecator instead of rolling our own: https://github.com/ManageIQ/linux_admin/pull/186/files
 
> But seriously... why can't Gemfile.dev.rb live in bundler.d/? That's where mine is and it works fine. I want to get rid of the code duplication.

Sure, once we deprecate it and give them sometime to move to `bundler.d` we can remove it.